easybuild/easyconfigs/j/jax/jax-0.2.18-foss-2020b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'jax'
+version = '0.2.18'
+
+homepage = 'https://pypi.python.org/pypi/jax'
+description = """Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and
+more"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+# source_urls = [PYPI_SOURCE]
+# sources = [SOURCE_TAR_GZ]
+
+source_urls = ['https://github.com/google/jax/archive/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+patches = [
+    'jax-0.2.18-correct_libraries.patch',
+]
+
+preinstallopts = 'python3 build/build.py --output_user_root=%(builddir)s --bazel_path="$EBROOTBAZEL/bin/bazel" && '
+preinstallopts += 'pip install dist/*.whl && '
```

Comment:
@deniskristak This takes part of building `jaxlib` from source, and hence there's no need to include it as a dependency/component, is that right?

cc @sassy-crick

Reply:
Correct

Reply:
please elaborate, I dont understand :D

Reply:
We don't need an extra dependency in EC on jaxlib as this EC installs jaxlib AND jax. That was @boegel question

Reply:
so no changes on this line required, right? :D

Reply:
Basically yes. Although I'd like to see jaxlib being separate (but in the same PythonBundle EC) I'll open a PR for PythonPackage to make that possible

Reply:
PythonPackage PR: https://github.com/easybuilders/easybuild-easyblocks/pull/2534
@boegel I think this would be great, especially as we can easily test this with this PR once adapted

@deniskristak Once you converted this into a PythonBundle (see https://github.com/easybuilders/easybuild-easyconfigs/pull/13622#discussion_r683396454 and the TF EC for an example and suggested versions) I can help you get this up with the new change if the description is not enough

Reply:
this should be resolved